<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_page</class>
 <widget class="QWidget" name="settings_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="widget_layout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="cabins_group">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Cabins</string>
     </property>
     <layout class="QVBoxLayout" name="cabins_group_layout">
      <item>
       <layout class="QHBoxLayout" name="cabins_num_layout">
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QLabel" name="num_cabins_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cabins Numbers:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QSpinBox" name="num_cabins_spinb">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="cabins_table">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Cabins</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Color Pickers</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Colors</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="operators_group">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Operators</string>
     </property>
     <layout class="QVBoxLayout" name="operators_layout">
      <item>
       <layout class="QHBoxLayout" name="num_operators_layout">
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QLabel" name="num_operators_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Operators Numbers:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QSpinBox" name="num_operators_spinb">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="operators_table">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Operators</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Names</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="schedule_group">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Working Hours</string>
     </property>
     <layout class="QHBoxLayout" name="schedule_layout">
      <item alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="min_hour_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QTimeEdit" name="min_hour_tedit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="maximumTime">
         <time>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="minimumTime">
         <time>
          <hour>6</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string notr="true">HH:mm</string>
        </property>
        <property name="time">
         <time>
          <hour>7</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="max_hour_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Finish:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QTimeEdit" name="max_hour_tedit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="minimumTime">
         <time>
          <hour>14</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string notr="true">HH:mm</string>
        </property>
        <property name="time">
         <time>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_pbtn">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
